draft-packages/loading-placeholder/KaizenDraft/LoadingPlaceholder/styles.scss

```diff
@@ -1,14 +1,15 @@
-@import "~@kaizen/design-tokens/sass/color";
 @import "~@kaizen/design-tokens/sass/color-vars";
 @import "~@kaizen/design-tokens/sass/border";
 @import "~@kaizen/component-library/styles/animation";
```

Comment:
Can we remove this import and swap the variables to `design-tokens/sass/animation-vars`? 
@kaizen/component-library/styles/animation